### DISCUSSION TITLE
Riattivazione in base al carico conosciuto e preimpostato

### DISCUSSION THREAD

**Discussion by Zaraffo** [Home-Assistant-Pro-Team/Power-Control-HomeAssistant#10] [Ideas]
Ciao, volevo inserire un suggerimento riguardante la gestione della riattivazione dei carichi, sarebbe utile inserire associato all' utilizzatore anche l'assorbimento max, o anche mantenere memorizzato l'ultimo valore del carico per cui si sia verificato il distacco, 
in modio tale che venga riattivato soltanto se la differenza tra la fornitura e il carico, più il consumo istantaneo sia al di sotto della soglia per cui verrà distaccato, così da evitare le ripetute riattivazioni e disattivazioni quando ancora l'assorbimento istantaneo è alto.
Esempio ho una fornitura di 3Kw e un carico di 1 Kw (scadabagno), e una soglia di riattivazione di 2900 W, io conoscendo (perchè preimpostato o ripescato dall'ultimo valore prima del distacco) l'assorbimento per ogni carico, non 
tenterò una riattivazione del carico finchè non sia soddisftatta la relazione: 
("fornitura massima" - "carico" + "potenza istantanea") < "soglia riattivazione"
(3000-1000 +2150) < 2900 = Non riattivo
(3000-1000+800) < 2900 = Riattivo

**Comment by marco-hacs** [comment]
Ottima idea... Come ho un minuto cercherò di implementare.

**Reply by marco-hacs** [reply]
E' stata implementata la richiesta. 

![image](https://github.com/Home-Assistant-Pro-Team/Power-Control-HomeAssistant/assets/62516592/21ed846b-dca1-49bb-8a5e-bdd1e83effaa)
